src/main/java/etgov/mof/pfmrt/conf/model/Organization.java

```diff
@@ -21,6 +21,7 @@
 	private String email;
 	private String telephone;
 	private boolean status;
+	private String organizationhead;
```

Comment:
this field has been removed

Reply:
ya but I was thinking, don't we need to capture his name? we need it when we penalize him